AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_GridAdded.py

```diff
@@ -119,35 +123,42 @@ def AtomEditorComponents_Grid_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.creation_redo, grid_entity.exists())
 
-        # 5. Enter/Exit game mode.
+        # 5. Grid Size changed (default is 32, changed to 64)
```

Comment:
Don't believe you need to call out the value changes here in the comment. Just leave it as "Grid Size changed"

Reply:
fixed